dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -653,21 +653,22 @@
 
                                 <ContentPresenter x:Name="ContentPresenter"
                                     Grid.Column="1"
+                                    Margin="0,0,16,0"
```

Comment:
>Margin="0,0,16,0" [](start = 36, length = 17)

Should we make this a resource so it can be light weight styled?

Reply:
lets track this as a separate issue. Thanks.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/issues/2288